returnn/torch/engine.py

```diff
@@ -913,9 +951,16 @@ def _save_model(self):
         tmp_filename = filename + ".tmp_write"
         if os.path.exists(tmp_filename):
             os.unlink(tmp_filename)
+        state_dict = self._pt_model.state_dict()
+        if self._ignore_param_set:
+            # Do some extra check that we don't save the ignored parameters.
```

Comment:
I was wondering if this would cause downstream ergonomics issues if the resulting model was used outside of RETURNN, where the preloading logic was less handily available, e.g. in ONNX, and if you then do suddenly end up depending on their values. I suppose this is a weird edge case though, because if so, just load these parameters normally in training and freeze them.

Reply:
Yea I would say this is up to the user. RETURNN should not restrict the user to do what he/she wants.